packages/volto/src/helpers/ScrollToTop/ScrollToTop.jsx

```diff
@@ -22,12 +28,30 @@ class ScrollToTop extends React.Component {
     children: PropTypes.node.isRequired,
   };
 
+  /**
+   * Used to indicate client-side rendering
+   * @memberof ScrollToTop
+   */
+  componentDidMount() {
+    this.isClientSide = true;
```

Comment:
@Faakhir30 I think the convention for this variable name is `isClient` and maybe you should use the component state for it :thinking: 

See for example: https://github.com/plone/volto/blob/main/packages/volto/src/components/theme/View/View.jsx#L114

Reply:
@avoinea Thanks for sharing the sample reference. I did think, IMO, that `isClient` wasn't required in my case, so I removed that variable in my recent commit. The reason is that `componentDidUpdate` never gets triggered during SSR; it is only invoked on the client side.